src/features/hotspots/setup/HotspotSetupPickHotspotScreen.tsx

```diff
@@ -15,6 +19,11 @@ const HotspotSetupPickHotspotScreen = () => {
     scannedDevices.length,
   ])
 
+  // Segment track for bluetooth scan
+  track(HotspotEvents.BLUETOOTH_SCAN_FINISHED, {
+    scanned_devices_count: scannedDevices.length,
```

Comment:
Would it be hard to include the time elapsed scanning?

Reply:
Now, the `elapsed_milliseconds` is included.